### ISSUE TITLE
Hugging Face Import does not work

### ISSUE THREAD

**Issue by yannickfunk** [hugofloresgarcia/audacity#75]
**Describe the bug**
When I click 'Add from Huggingface', enter 'hugggof/openl3-labeler-w-timestamps' and press enter,
a directory 'hugggof_openl3-labeler-w-timestamps' is created inside the 'deeplearning-models' directory but not filled with content. Also the imported model does not show up in the deep learning effect selection dialogue

**To Reproduce**
1. Go to 'Deep Learning Effect'
2. Click on 'Add From HuggingFace'
3. Enter 'hugggof/openl3-labeler-w-timestamps'

**Expected behavior**
The model should be downloaded and showing up in the selection dialogue

**Additional information (please complete the following information):**
 - OS: Windows 10 Pro 19042.1320
 - Audacity 3.2.0-alpha-20211028

**Comment by hugofloresgarcia** [comment]
Hi! Could you send me  the error logs, located in `Help->Diagnostics->Show Log`?

**Comment by yannickfunk** [comment]
```
16:34:03: Fetching model card from https://huggingface.co/hugggof/openl3-labeler-w-timestamps/resolve/main/metadata.json
16:34:03: Fetching model card from https://huggingface.co/hugggof/ConvTasNet-DAMP-Vocals/resolve/main/metadata.json
16:34:03: Fetching model card from https://huggingface.co/hugggof/ConvTasNet_Libri3Mix_sepnoisy_16k/resolve/main/metadata.json
16:34:03: Fetching model card from https://huggingface.co/hugggof/ConvTasNet_WHAM_sepclean/resolve/main/metadata.json
16:34:03: Fetching model card from https://huggingface.co/hugggof/ConvTasNet_Libri1Mix_enhsignle_16k/resolve/main/metadata.json
16:34:03: Fetching model card from https://huggingface.co/hugggof/demucs_extra/resolve/main/metadata.json
16:34:03: https://huggingface.co/hugggof/ConvTasNet-DAMP-Vocals/resolve/main/metadata.json
16:34:03: https://huggingface.co/hugggof/ConvTasNet_Libri1Mix_enhsignle_16k/resolve/main/metadata.json
16:34:03: https://huggingface.co/hugggof/ConvTasNet_Libri3Mix_sepnoisy_16k/resolve/main/metadata.json
16:34:03: https://huggingface.co/hugggof/demucs_extra/resolve/main/metadata.json
16:34:03: https://huggingface.co/hugggof/ConvTasNet_WHAM_sepclean/resolve/main/metadata.json
16:34:03: https://huggingface.co/hugggof/openl3-labeler-w-timestamps/resolve/main/metadata.json
16:34:20: Fetching model card from https://huggingface.co/hugggof/openl3-labeler-w-timestamps/resolve/main/metadata.json
16:34:21: https://huggingface.co/hugggof/openl3-labeler-w-timestamps/resolve/main/metadata.json

```

**Comment by hugofloresgarcia** [comment]
Hmm. I couldn't seem to reproduce the issue on my Windows or Mac machines. Could you try the latest binary (updated today) here? https://interactiveaudiolab.github.io/project/audacity. Let me know if this fixes it. Thanks!

**Comment by yannickfunk** [comment]
With the latest binary, the problem still persists. C:\Users\Yannick\AppData\Roaming\audacity\deeplearning-models\hugggof_openl3-labeler-w-timestamps gets created, but is empty

**Comment by tin2tin** [comment]
I think the label models needs to be installed through the deep learning analyse pop up(not the one found in the effects menu.

**Comment by yannickfunk** [comment]
Yes that works, thank you, is that documented somewhere?

**Comment by hugofloresgarcia** [comment]
Oh, yes, so sorry! I missed that when reading the steps to reproduce. Thanks @tin2tin!

**Comment by tin2tin** [comment]
I properly didn't explain it well, but I attempted to do so here: #67

**Comment by hugofloresgarcia** [comment]
yeah, I think the UI could have done a better job at letting you know that you should be using the other effect: #76. @aldo-aguilar  could you work on a fix?

**Comment by aldo-aguilar** [comment]
sounds good thanks for looping me in.

**Comment by bryan-pardo** [comment]
One fix to this would be to fix the video that is shown on startup when you first install. Right now, when you first install, you get a help page that says "what's new in Audacity 3.2.xxxx" with a link to a video for Audacity 3.1.   It might be good to have a link to a video for how to use the features we introduce in 3.2.

**Comment by hugofloresgarcia** [comment]
closing this, as it will be fixed when #76 gets fixed.
